### PR TITLE
Fix cudnn filename in configure script.

### PR DESCRIPTION
**Pull request by gunan**

### CODE REVIEW

File: tensorflow/core/platform/default/build_config.bzl [hunk 1]
```diff
@@ -7,7 +7,7 @@ load("//tensorflow:tensorflow.bzl", "if_not_mobile")
 # configure may change the following lines
 WITH_GCP_SUPPORT = False
 WITH_HDFS_SUPPORT = False
-WITH_JEMALLOC = True
```
Comment (rmlarsen) [inline tensorflow/core/platform/default/build_config.bzl]: You didn't mention this change in the description. Is it intentional?

Comment (gunan) [inline tensorflow/core/platform/default/build_config.bzl]: It is not. It is an artifact of trying to locally test the change.
Removed.

**Review by rmlarsen — APPROVED** [review]

**Comment by rmlarsen** [comment]
@gunan looks like agent died again for the mac build. Want to just kick off that one?

**Comment by rmlarsen** [comment]
@tensorflow-jenkins test this please

**Comment by gunan** [comment]
Sorry for the delay, either way, I will investigate why agents are going offline this much today.

**Pull request merged**